README.rst

```diff
@@ -203,46 +280,14 @@ The tracing wrapper creates a span for the wrapper handler. This span has the fo
 |                                  | ‘python-lambda-wrapper’          |
 +----------------------------------+----------------------------------+
 
-Sending custom metric from the Lambda function
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. code:: python
 
-    import signalfx_lambda
+Steps 7 & 8 don't have (and shouldn't) be done by most users. This is rather info for contributors who want to test changes they do in the wrapper and/or publish a new version. As such, I would vote to have them in last separate section with a title "For contributors" or "For maintainers" or just "Additional info". The metrics and dimensions and tags which are now last are probably of interest to most users, but this two steps are not.
 
-    # sending application_performance metric with value 100 and dimension abc:def
-    signalfx_lambda.send_gauge('application_performance', 100, {'abc':'def'})
 
-    # sending counter metric with no dimension
-    signalfx_lambda.send_counter('database_calls', 1)
+Test configurations locally 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For users who are interested in testing changes to a wrapper or in publishing a new version, run the following commmand in your command line: 
```

Comment:
I would remove "or in publishing a new version" so that this sentence just becomes:
"For users who are interested in testing changes to a wrapper, run the following command in your command line:"

Reply:
This has been updated.